src/nvme/ioctl.h

```diff
@@ -5056,12 +5056,68 @@ static inline int nvme_lm_track_send(nvme_link_t l, __u16 mos, __u16 cdqid, __u8
 /**
  * nvme_lm_migration_send() - Migration Send command
  * @l:		Link handle
- * @args:	&struct nvme_lm_migration_send_args argument structure
+ * @offset:	Offset: This field specifies the offset, in bytes, within the data available to be
```

Comment:
Let's rename it to `lpo`, so it matches with the other commands. And it should be before `data`.

Reply:
yes, let's follow the spec here closely. That avoids misunderstanding (I thought `offset` was `lpo` but it isn't).

The arg should be be of type `u64` and should be called `cso`.

Reply:
Noted.